src/components/review/AssignTechnicalReview.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, makeStyles, TextField } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import React, { useState } from 'react';
+
+import { BasicUserDetails, Proposal, UserRole } from 'generated/sdk';
+import { useUsersData } from 'hooks/user/useUsersData';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+import withConfirm, { WithConfirmType } from 'utils/withConfirm';
+
+type AssignTechnicalReviewProps = {
+  proposal: Proposal;
+  onProposalUpdated: (proposal: Proposal) => void;
+  confirm: WithConfirmType;
+};
+
+const getName = (user: BasicUserDetails | undefined) =>
```

Comment:
This `getName` function can be renamed to something like `getFullName` and moved to helper functions and reused in many places as we need full name in many components.